samples/AppSecretsConfig/README.md

```diff
@@ -0,0 +1,251 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-app-configuration
+- azure-key-vault
+urlFragment: app-secrets-configuration
+name: Configure Applications with App Configuration and Key Vault
+description: Improve application startup performance when configuring secrets in Key Vault using Azure App Configuration.
+---
+
+# Configure Applications with App Configuration and Key Vault
+
+The [Azure Key Vault][keyvault_overview] provides a great service to store application secrets and use them to configure your applications; however, with relatively [low limits][keyvault_limits] some applications referencing Key Vault with a lot of secrets or even clustered applications referencing fewer secrets can get rate limited and experience slow startup performance. You should instead use [Azure App Configuration][appconfig_overview] to store non-secrets and define references to secrets in Key Vault to reduce the number of calls made to Key Vault.
+
+Examples of non-secrets that should be stored in App Configuration:
+
+* Client application IDs
+* IP addresses
+* Service endpoints
+* Service configuration parameters
+* Usernames
+
+Examples of secrets that should be stored in Key Vault:
+
+* Client application secrets
+* Connection strings
+* Passwords
+* Shared access keys
+* SSH keys
+
+## Getting started
```

Comment:
Should we throw a prerequisites section in here and call out the Azure CLI and .NET SDK?

Reply:
I reformatted this a bit and did reference .NET and the az CLI, but didn't go with an explicit pre-reqs section. Just felt it didn't flow right with how we do overviews and such, but I think it still calls it out obviously.